pandas/tseries/tools.py

```diff
@@ -214,14 +218,18 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         Return UTC DatetimeIndex if True (converting any tz-aware
         datetime.datetime objects as well).
     box : boolean, default True
+
         - If True returns a DatetimeIndex
         - If False returns ndarray of values.
+
```

Comment:
For these, the extra blank space should normally not be needed (as I explained above). Did you get warnings for these as well?